src/egit.c

```diff
@@ -352,4 +362,12 @@ void egit_init(emacs_env *env)
 
     // Revparse
     DEFUN("libgit-revparse-single", revparse_single, 2, 2);
+
+    // Branch
+    DEFUN("libgit-branch-create", branch_create, 3, 4);
+    DEFUN("libgit-branch-create-from-annotated", branch_create_from_annotated, 3, 4);
+    DEFUN("libgit-branch-delete", branch_delete, 2, 2);
+    DEFUN("libgit-branch-is-checked-out", branch_is_checked_out, 2, 2);
+    DEFUN("libgit-branch-is-head", branch_is_head, 2, 2);
+    DEFUN("libgit-lookup", branch_lookup, 2, 3);
```

Comment:
Should these be libgit-branch-head-p and libgit-branch-checkout-out-p? As per the readme.

Reply:
Missed that, thanks